.fastlane/Fastfile

```diff
@@ -130,10 +140,12 @@ platform :ios do
 
     slack(
       slack_url: ENV["SLACK_WEBHOOK"],
-      message: "Successfully uploaded" +
-        " v" + get_version_number(xcodeproj: "Kickstarter.xcodeproj", target: "Kickstarter-iOS") +
-        " (" + get_build_number(xcodeproj: "Kickstarter.xcodeproj") + ")" +
-        " of the Kickstarter iOS app to iTunes Connect 🚀"
+      message: slack_message(
+        "production",
```

Comment:
Hmm I wonder if this would be clearer as "App Store" or "iTunes Connect"?

Reply:
Oh sure, what do you suggest: `App Store`, `Alpha`, `Beta`?

Seems like Apple has _finally_ renamed _iTunes Connect_ to _App Store_ 🍾